Dockerfile

```diff
@@ -0,0 +1,25 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.12.6-slim
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+# Install minimal OS deps
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    curl \
```

Comment:
What do we need curl for at this stage ?

Reply:
I was testing something from the container. Removed.